openlibrary/plugins/openlibrary/js/search.js

```diff
@@ -68,3 +105,55 @@ export function initSearchFacets() {
         less($(this).data('header'), start_facet_count, facet_inc);
     });
 }
+
+/**
+ * A successful response from the partials request
+ *
+ * @typedef {Object} PartialsResponse
+ * @property {string} title - The new title for this page
+ * @property {string} sidebar - HTML string for the facets sidebar
+ * @property {string} [activeFacets] - HTML string for the selected facets
+ */
+/**
+ * Using the given search parameters, makes call for a partially rendered facet affordances
+ * and the new title for the page.
+ *
+ * @param {Object} param
+ * @returns {Promise<PartialsResponse>}
+ *
+ * @throws Error when `/partials` response is not in 200-299 range.
+ */
+function fetchPartials(param) {
+    const data = {
+        param: param,
+        path: location.pathname,
+        query: document.location.search
```

Comment:
Be consistent with above

```suggestion
        query: location.search
```